.github/workflows/ci.yml

```diff
@@ -109,4 +110,46 @@ jobs:
             echo "No integration tests directory found; skipping."
           fi
 
-
+      - name: Generate coverage report
+        if: success()
+        working-directory: mediawiki
+        run: |
+          mkdir -p build/logs
+          EXT_PATH="$(pwd)/extensions/LabkiPackManager"
+          echo "wfLoadExtension( 'LabkiPackManager' );" >> LocalSettings.php
+
+          cat > "$EXT_PATH/tests/phpunit/coverage.xml" <<XML
+          <?xml version="1.0" encoding="UTF-8"?>
+          <phpunit bootstrap="tests/phpunit/bootstrap.php">
+            <coverage>
+              <include>
+                <directory suffix=".php">$EXT_PATH/includes</directory>
+                <directory suffix=".php">$EXT_PATH/maintenance</directory>
+              </include>
+            </coverage>
+          </phpunit>
+          XML
```

Comment:
seems like this could just be in the repo if we have to write it to run the tests?